src/buildHooks.ts

```diff
@@ -118,7 +121,16 @@ export function buildHooks<Definitions extends EndpointDefinitions>({
   }
 
   function buildQueryHook(name: string): QueryHook<any> {
-    return (arg: any, { refetchOnMountOrArgChange = false, skip = false, pollingInterval = 0 } = {}) => {
+    return (
+      arg: any,
+      {
+        refetchOnMountOrArgChange = config.refetchOnMountOrArgChange,
+        refetchOnFocus = config.refetchOnFocus,
+        refetchOnReconnect = config.refetchOnReconnect,
```

Comment:
Assuming the global config on `refetchOnMountOrArgChange` & co can be changed at some point in time using the global config state, shouldn't these default options be selected from the config state instead of the initial config object?
So, with a `useSelector` in the hook body.

Or: does it even make sense to set to the default values here instead of just applying whatever is in the config state when "reading" the value later in the middleware?

Reply:
I've kept them here for now, but I'll see about removing them in a cleanup pass.